radix-transaction-scenarios/src/scenarios/account_locker.rs

```diff
@@ -0,0 +1,777 @@
+                                .create_fungible_resource(
+                                    OwnerRole::None,
+                                    true,
+                                    0,
+                                    FungibleResourceRoles {
+                                        mint_roles: mint_roles! {
+                                            minter => rule!(allow_all);
+                                            minter_updater => rule!(deny_all);
+                                        },
+                                        burn_roles: burn_roles! {
+                                            burner => rule!(allow_all);
+                                            burner_updater => rule!(deny_all);
+                                        },
+                                        ..Default::default()
+                                    },
+                                    Default::default(),
+                                    None,
+                                )
+                                .create_non_fungible_resource::<[(NonFungibleLocalId, ()); 0], _>(
+                                    OwnerRole::None,
+                                    NonFungibleIdType::Integer,
+                                    true,
+                                    NonFungibleResourceRoles {
+                                        mint_roles: mint_roles! {
+                                            minter => rule!(allow_all);
+                                            minter_updater => rule!(deny_all);
+                                        },
+                                        burn_roles: burn_roles! {
+                                            burner => rule!(allow_all);
+                                            burner_updater => rule!(deny_all);
+                                        },
+                                        ..Default::default()
+                                    },
+                                    Default::default(),
+                                    None,
+                                )
+                        },
+                        vec![],
+                    )
+                },
+                |_, _, state, result| {
+                    state
+                        .fungible_resource
+                        .set(result.new_resource_addresses()[0]);
+                    state
+                        .non_fungible_resource
+                        .set(result.new_resource_addresses()[1]);
+                    Ok(())
+                },
+            )
+            .successful_transaction(|core, config, state| {
+                core.next_transaction_with_faucet_lock_fee(
+                    "account-locker-setting-up-account-deposit-rules",
+                    |builder| {
+                        builder
+                            .call_method(
+                                config.user_account1.address,
```

Comment:
For clarity when it comes to looking at the named addresses, I'd prefer this account to be named, e.g. `account_with_deny_resource_preference`